domain/comment.go

```diff
@@ -0,0 +1,17 @@
+package domain
+
+import "time"
+
+type Comment struct {
+	ID        string    `json:"id" yaml:"id"`
+	AppealID  string    `json:"appeal_id" yaml:"appeal_id"`
```

Comment:
Can we look into having this association in a way that it can be plugged into any resource worldwide?

For eg: Using a resource URN?

Reply:
will change this field to `ParentID`

Reply:
this will make us lose the benefit of using foreign key and also need to add `parent_type` column to avoid conflict between different parent type with the same ID. Shall we continue with this?